span/daq.py

```diff
@@ -58,31 +62,31 @@ def _addInputChannels(self, task, channels):
             task.ai_channels.add_ai_voltage_chan(channel)
     
     
-    def _configureChannelTimings(self, task, samples):
+    def _configureChannelTimings(self, task: dx.task.Task, samples: int) -> None:
         """
         Set the correct timings for task based on number of samples
         """
-        assert self.samplerate, "Samplerate should be set first!"
+        assert self.samplerate, "Samplerate should be set first."

```

Comment:
In the name of consistency, we should probably change this to `assert not (self.samplerate is None), "Samplerate should be set first."`